reflex/components/radix/primitives/slider.py

```diff
@@ -0,0 +1,170 @@
+"""Radix slider components."""
+
+from typing import Any, Dict, Literal
+
+from reflex.components.component import Component
+from reflex.components.radix.primitives.base import RadixPrimitiveComponent
+from reflex.style import Style
+from reflex.vars import Var
+
+LiteralSliderOrientation = Literal["horizontal", "vertical"]
+LiteralSliderDir = Literal["ltr", "rtl"]
+
+
+class SliderComponent(RadixPrimitiveComponent):
+    """Base class for all @radix-ui/react-slider components."""
+
+    library = "@radix-ui/react-slider@^1.1.2"
+
+
+class SliderRoot(SliderComponent):
+    """The Slider component comtaining all slider parts."""
+
+    tag = "Root"
+    alias = "RadixSliderRoot"
+
+    default_value: Var[list[int]]
+
+    value: Var[int]
+
+    name: Var[str]
+
+    disabled: Var[bool]
+
+    orientation: Var[LiteralSliderOrientation]
+
+    dir: Var[LiteralSliderDir]
+
+    inverted: Var[bool]
+
+    min: Var[int]
+
+    max: Var[int]
+
+    step: Var[int]
+
+    min_steps_between_thumbs: Var[int]
+
+    def get_event_triggers(self) -> Dict[str, Any]:
+        """Event triggers for radix slider primitive.
+
+        Returns:
+            The triggers for event supported by radix primitives.
+        """
+        return {
+            **super().get_event_triggers(),
+            "on_value_change": lambda e0: [e0],  # trigger for all change of a thumb
+            "on_value_commit": lambda e0: [e0],  # trigger when thumb is released
+        }
+
+    def _apply_theme(self, theme: Component):
+        self.style = Style(
+            {
+                "position": "relative",
+                "display": "flex",
+                "align-items": "center",
+                "user-select": "none",
+                "touch-action": "none",
```

Comment:
for consistency with other parts of reflex, should we use snake case here?

```suggestion
                "align_items": "center",
                "user_select": "none",
                "touch_action": "none",
```